features/steps/project/source/browse_files.rb

```diff
@@ -85,6 +89,15 @@ class Spinach::Features::ProjectSourceBrowseFiles < Spinach::FeatureSteps
     click_button 'Remove file'
   end
 
+  step 'I click on "Replace"' do
+    click_button 'Replace'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.